templates/overview.html.twig

```diff
@@ -0,0 +1,39 @@
+<html>
+
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  <style>
+      .w3-theme {
+          color:#fff !important; 
+          background-color:#40114f !important;
+      }
+      .disabled {
+          opacity: 0.6;
+      }
+  </style>
```

Comment:
Inline CSS is discouraged.  I'm going to let it slide here but I'm adding a new issue to implement better asset management to help with this kind of thing.

Reply:
Issue #16 created to address this